src/float/add.rs

```diff
@@ -181,117 +181,30 @@ macro_rules! add {
     }
 }
 
+// NOTE(cfg) These are disabled for gnueabihf due to 
+// https://github.com/rust-lang-nursery/compiler-builtins/issues/90
+#[cfg(not(gnueabi))]
```

Comment:
Right now, cfg(gnueabihf) works but cfg(gnueabi) doesn't. You'll have to update the build.rs to make Cargo pass this cfg when the target is arm-unknown-linux-gnueabi. See https://github.com/rust-lang-nursery/compiler-builtins/blob/master/build.rs#L428